lighthouse-core/audits/seo/font-size.js

```diff
@@ -243,7 +243,7 @@ class FontSize extends Audit {
    * @return {Promise<LH.Audit.Product>}
    */
   static async audit(artifacts, context) {
-    if (!artifacts.TestedAsMobileDevice) {
+    if (context.settings.formFactor === 'desktop') {
```

Comment:
seems intermittent whether `!== 'mobile'` or `=== 'desktop'` is used. do you have a preference or proposal for when to use one vs. the other?

I can't think of a case where we'll be adding a third at the moment, so might not matter for awhile

Reply:
should be using `=== desktop` most everywhere. whenever they are comments they discuss desktop, so that's what i aligned against.